### DISCUSSION TITLE
Translatable API

### DISCUSSION THREAD

**Discussion by JonathanSchndr** [directus/directus#17485] [Feature Requests]
### Summary

Users speak different languages. However, the API always responds in English. Solution: The API interprets the Accept-Language Header or possibly a parameter to answer the user in the correct language.

### Basic Example

1. the user sends a request to POST /auth/password/request
2. the API reads the accept-language and checks if the response/action has areas that can be translated and then returns them in the correct language
3. for example a mail would be sent in the correct language with the correct translation in the subject line

... or errors or login erros could be returned directly in the correct language. So the frontend application saves to intercept them or to translate them again. This translatable content will first run through the language files, but can also be created again by the user in the Directus backend.

### Motivation

The idea came up because there is currently no way to change the Password Reset Request subject. Although you can change the content via the template, the subject is hardwired in the code. 

https://github.com/directus/directus/blob/cc7d3db099eb9148de03c1ed8ac657373ad07429/api/src/controllers/auth.ts#L158 
https://github.com/directus/directus/blob/cc7d3db099eb9148de03c1ed8ac657373ad07429/api/src/services/users.ts#L390

In addition, there are many users who own this desire. Only here thought a little larger.

https://github.com/directus/directus/pull/7372
https://github.com/directus/directus/pull/16673
https://github.com/directus/directus/pull/15590

### Detailed Design

Must be discussed

### Requirements List

Must be discussed

### Drawbacks

Must be discussed

### Alternatives

If this is not desired, it would at least be good to get an ENV as a quick solution to at least adjust the subject for Password Reset Request.

### Adoption Strategy

There should be a switch that activates this translatable API. Only if this is actively set by the user, this function is active. This ensures that users are not confused by translatable response - default english.

### Unresolved Questions

This should be the start of a discussion. Many questions and wishes are still open. I am happy about the ideas from the team and would like to support the implementation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
